storage/client_split_test.go

```diff
@@ -76,7 +76,9 @@ func TestStoreRangeSplitAtIllegalKeys(t *testing.T) {
 		keys.Meta1Prefix,
 		keys.MakeKey(keys.Meta1Prefix, []byte("a")),
 		keys.MakeKey(keys.Meta1Prefix, proto.KeyMax),
+		keys.Meta2KeyMax,
```

Comment:
> I can either remove it entirely

What is "it"? Anyway it'd be good to reduce duplication while you're here, whichever way you decide to do it.